apps/web/src/components/RegisterForm/RegisterForm.tsx

```diff
@@ -36,56 +32,189 @@ export default function RegisterForm(): JSX.Element {
       formData.entries()
       // eslint-disable-next-line
     ) as any;
-    const { email, password } = formValue;
+    const { email, password, confirmPassword } = formValue;
 
     assert(recaptchaRef.current, 'ReCAPTCHA has not loaded');
-    const token = await recaptchaRef.current.executeAsync();
+    const token = recaptchaRef.current;
 
     assert(token, 'Missing ReCAPTCHA token');
 
-    await api.post('/users/create', JSON.stringify(formValue));
-    await logIn(email, password);
-    history.push('/calendar');
+    if (password !== confirmPassword) {
+      setDisplayError('Passwords do not match!');
+    } else {
+      setDisplayError('');
+      await await api
```

Comment:
await await is not correct.

Will fix this real quick.

Reply:
Not sure what the heck happened there, thank you!